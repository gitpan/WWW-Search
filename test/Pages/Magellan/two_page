http://www.mckinley.com/search.gw?look=magellan&start=0&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=10&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=20&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=30&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=40&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=50&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=60&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=70&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=80&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=90&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=100&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=110&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=120&search=%2BISI+%2Bdivision&
http://www.mckinley.com/search.gw?look=magellan&start=130&search=%2BISI+%2Bdivision&
