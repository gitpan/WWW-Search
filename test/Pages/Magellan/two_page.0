<HTML>
<HEAD><TITLE>Search Results</TITLE></HEAD>
<BODY BGCOLOR="#ffffff">

 <CENTER>
 <TABLE BORDER=0 CELLPADDING=0 CELLSPACING=2>
  <TR>
   <TD ALIGN=LEFT VALIGN=top ROWSPAN=4>
    <A HREF="http://magellan.excite.com/"><IMG SRC="/img/magellan/mglobe.gif" BORDER=0 WIDTH=49 HEIGHT=41 ALIGN=left></A>
   </TD>
   <TD ALIGN=middle VALIGN=top ROWSPAN=4>
    <A HREF="http://magellan.excite.com/"><IMG SRC="/img/magellan/magellan.gif" BORDER=0 WIDTH=224 HEIGHT=41 ALIGN=middle ALT="Magellan Internet Guide"></A><BR>
    <IMG SRC="/img/magellan/white.gif" HEIGHT=4 WIDTH=1><BR>
   </TD>
   <TD WIDTH=4 ROWSPAN=4></TD>
   <TD HEIGHT="2" COLSPAN=3 NOWRAP></TD>
  </TR>

  <FORM ACTION="/search.gw" METHOD=get>
   <TR>
    <TD ALIGN=left VALIGN=top COLSPAN=3 NOWRAP>
     <INPUT NAME=search VALUE="+ISI +division" SIZE=30>
     <INPUT TYPE=IMAGE SRC="/img/magellan/searchbut.gif" BORDER=0 ALIGN="top" VALUE="Search Magellan" WIDTH=67 HEIGHT=30>
    </TD>
   </TR>

   <TR>
    <TD ALIGN=left NOWRAP>
     <INPUT TYPE=radio NAME=c VALUE=web  CHECKED>     <FONT SIZE="-1">The Entire Web</FONT>
     <INPUT TYPE=hidden NAME=look VALUE=magellan>
    </TD>
    <TD WIDTH=3 ROWSPAN=2></TD>
    <TD ALIGN=left>
     <INPUT TYPE=radio NAME=c VALUE=web.green >     <FONT SIZE="-1" COLOR=#008C05>Green Light</FONT>
     <FONT SIZE="-1"> sites only</FONT>
    </TD>
   </TR>

   <TR>
    <TD ALIGN=left></TD>

    <!-- End of search and collection section -->

    <TD ALIGN=left><FONT SIZE="-1">
     <A HREF="/magellan/Info/advancedtips.html">Search Tips and Help</A>
    </FONT></TD>
   </TR>

   <input type=hidden name=sorig value=rpage>
  </FORM>
 </TABLE>
 </CENTER>

<P>


 <CENTER>
      <!-- Ad Start -->
<!-- Ad param svcpage=/search.gw keyword=isi+division svccobrand=magellan svcname=xsr ip=147.81.3.197 user_agent=WWW::Search/2.03 -->
<a href="http://www.excite.com/relocate/ad=90760_casinosunrise050799_suncopy.gif|pk=2683256;http://www.casino-sunrise.com/cgi-bin/englsun.pl?301550" onMouseOver="window.status='Click Here !';return true" language="Javascript"><img src="http://image.eimg.com/ads/90760_casinosunrise050799_suncopy.gif" ALT="Click Here !" HEIGHT=60 WIDTH=468 BORDER=1><CENTER><FONT SIZE=-1 FACE=geneva,arial COLOR=0000FF> <B> Click Here ! </B></FONT></CENTER></a>
<!-- Ad Stop -->

   </CENTER>

<BLOCKQUOTE>

<B>140</B> results returned, ranked by relevance.

<!-- IF statement for Green Light text -->





<FORM ACTION="/search.gw" METHOD=get>
    <INPUT TYPE=hidden NAME=search VALUE="+ISI +division">
    <INPUT TYPE=hidden NAME=look VALUE=magellan>
          <INPUT TYPE=hidden NAME=start VALUE=0>
    <INPUT TYPE=hidden NAME=collection VALUE=web>
            
<!--search results-->

   <HR>
  <B><A HREF="http://www.isi.norgren.com/">ISI Norgren - automotive division </A></B>&nbsp;&nbsp;&nbsp;
  <B>67%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=812803&ctag=37AE6922&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      This page is designed to be viewed by a browser which supports Frames.  This text will be shown by browsers which do not support the Frames extension.   
    <BR>
  <I> <A HREF="http://www.isi.norgren.com/">http://www.isi.norgren.com/</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.procite.com/welcome.html">ProCite and Reference Manager and Reference Web Poster, Bibliographic ...</A></B>&nbsp;&nbsp;&nbsp;
  <B>62%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1182452&ctag=37AE6922&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      Collect references from online and Web databases Organize your personal reference collection easily Create bibliographies in one step Share references on a network or on the Internet  
    <BR>
  <I> <A HREF="http://www.procite.com/welcome.html">http://www.procite.com/welcome.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.kfa-juelich.de/isi/isi_e.html">Institute for Thin Film and Ion Technology</A></B>&nbsp;&nbsp;&nbsp;
  <B>61%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1920695&ctag=37AE6922&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      Interna (For members of the center only)  Member of the Hermann von Helmholtz Association of National Research Centres (HGF)  deutsch/ english  Oscillator of a Josephson junction array coupled to a Josephson mixer   
    <BR>
  <I> <A HREF="http://www.kfa-juelich.de/isi/isi_e.html">http://www.kfa-juelich.de/isi/isi_e.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.ast.cam.ac.uk/~ioalib/otherlit.html">Institute of Astronomy</A></B>&nbsp;&nbsp;&nbsp;
  <B>59%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=829069&ctag=37AE6922&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      BIDS ISI Science Citation Index.  Coverage 4,500 journals in all scientific subjects, 1981 to date (abstracts from 1991).  This service requires a password.   
    <BR>
  <I> <A HREF="http://www.ast.cam.ac.uk/~ioalib/otherlit.html">http://www.ast.cam.ac.uk/~ioalib/otherlit.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.isibrno.cz/~hanca/index.html">Homepage Hana Kucharova, ISI Brno</A></B>&nbsp;&nbsp;&nbsp;
  <B>59%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=787270&ctag=37AE6922&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      born in Boskovice, Czech Republic, in 1971  Ing. degree in Electrotechnical Engineering (Electrotechnology), Technical University Brno, 1994 PhD student at Technical University Brno, Faculty of Electrotechnical Engineering and Computer Science external PhD student at the Institute of Scientific Instruments, Acad. of Sc.   
    <BR>
  <I> <A HREF="http://www.isibrno.cz/~hanca/index.html">http://www.isibrno.cz/~hanca/index.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.hscc.net/">Home</A></B>&nbsp;&nbsp;&nbsp;
  <B>56%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=394616&ctag=37AE6922&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      High Speed Connectivity Consortium Capable of gigabit end user connections  Using end-to-end optical fiber connections Or mostly fiber-only connections   
    <BR>
  <I> <A HREF="http://www.hscc.net/">http://www.hscc.net/</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.isi-bltv.co.uk/ricochet/services.htm">Services</A></B>&nbsp;&nbsp;&nbsp;
  <B>56%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=606353&ctag=37AE6922&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      Established for over 10 years, the Ricochet Group opened up its first operation in the heart of Slough.  Since then we have undergone rapid expansion in order to attain our goal of developing an extensive network of offices throughout the UK mainland.  
    <BR>
  <I> <A HREF="http://www.isi-bltv.co.uk/ricochet/services.htm">http://www.isi-bltv.co.uk/ricochet/services.htm</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.isibrno.cz/~malczyk/index.html">Homepage Roman Malczyk, ISI Brno</A></B>&nbsp;&nbsp;&nbsp;
  <B>55%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=54196&ctag=37AE6922&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      born on April 27, 1972, in Karvina, Czech Republic  Ing. degree with distinction and the Rector's Award in Electrotechnical Engineering (Cybernetics, Biocybernetics), Technical University Brno, 1995   
    <BR>
  <I> <A HREF="http://www.isibrno.cz/~malczyk/index.html">http://www.isibrno.cz/~malczyk/index.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://inform.bnl.gov/RESLIB/reslib.html">Research Library at BNL</A></B>&nbsp;&nbsp;&nbsp;
  <B>55%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1458549&ctag=37AE6922&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      Welcome to the Research Library at Brookhaven National Laboratory Information Services Division - Brookhaven National Laboratory  Research Library  (See also: Inform System)   
    <BR>
  <I> <A HREF="http://inform.bnl.gov/RESLIB/reslib.html">http://inform.bnl.gov/RESLIB/reslib.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.hindustantimes.com/">The Hindustan Times</A></B>&nbsp;&nbsp;&nbsp;
  <B>54%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1259084&ctag=37AE6922&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      FRONT PAGE  GOURMET FARE  TODAY'S POLL  HT INVESTOR  "CALL INDIA FROM US: 39 CENTS/MIN"  ABOUT US  An injured soldier being airlifted from Ghumri  Pak backtracks, links pullout to Siachen Backtracking from its commitment to US President Bill Clinton, Pakistan has made it conditional that infiltrators would withdraw from Kargil only if India agreed to vacate the areas, including Siachen.   
    <BR>
  <I> <A HREF="http://www.hindustantimes.com/">http://www.hindustantimes.com/</A></I>
  <BR>
 <HR>
</FORM>

<FORM ACTION="/search.gw" METHOD=get>
             <INPUT TYPE=submit NAME=next VALUE="Next Results">
       <INPUT TYPE=hidden NAME=search VALUE="+ISI +division">
    <INPUT TYPE=hidden NAME=look VALUE=magellan>
       <INPUT TYPE=hidden NAME=start VALUE=0>
    <INPUT TYPE=hidden NAME=collection VALUE=web>
          </FORM>


Try your search on:&nbsp;
  <A HREF="http://www.excite.com/search.gw?s=%2Bisi+%2Bdivision"><B>Excite</B></A>


</BLOCKQUOTE>
<CENTER>

<P><CENTER>

      <!-- Ad Start -->
<a href="http://www.excite.com/relocate/ad=90760_casinosunrise050799_suncopy.gif|pk=2683256;http://www.casino-sunrise.com/cgi-bin/englsun.pl?301550" onMouseOver="window.status='Click Here !';return true" language="Javascript"><img src="http://image.eimg.com/ads/90760_casinosunrise050799_suncopy.gif" ALT="Click Here !" HEIGHT=60 WIDTH=468 BORDER=1><CENTER><FONT SIZE=-1 FACE=geneva,arial COLOR=0000FF> <B> Click Here ! </B></FONT></CENTER></a>
<!-- Ad Stop -->

  
</CENTER><P>

<FORM ACTION="/search.gw" METHOD=get>
 <TABLE BORDER=0 CELLPADDING=0 CELLSPACING=2>
  <TR>
   <TD ALIGN=left VALIGN=top COLSPAN=3 NOWRAP>
    <INPUT NAME=search VALUE="+ISI +division" SIZE=30>
    <INPUT TYPE=IMAGE SRC="/img/magellan/searchbut.gif" BORDER=0 ALIGN="top" VALUE="Search Magellan" WIDTH=67 HEIGHT=30>
   </TD>
  </TR>
  <TR>
   <TD ALIGN=left>
    <INPUT TYPE=radio NAME=c VALUE=web
     CHECKED> 
    <FONT SIZE="-1">The Entire Web</FONT>    <INPUT TYPE=hidden NAME=look VALUE=magellan>
   </TD>
   <TD WIDTH=3 ROWSPAN=2></TD>
   <TD ALIGN=left>
    <INPUT TYPE=radio NAME=c VALUE=web.green
    >    <FONT SIZE="-1" COLOR=#008C05>Green Light</FONT>
    <FONT SIZE="-1"> sites only</FONT>
   </TD>
  </TR>
  <TR>
   <TD ALIGN=left></TD>
   <TD ALIGN=left><FONT SIZE=-1>
    <A HREF="/magellan/Info/advancedtips.html">Search Tips and Help</A>
   </FONT></TD>
  </TR>
 </TABLE>
 <input type=hidden name=sorig value=rpage>
</FORM>


<P><BR>
<CENTER>
<FONT FACE="arial,helvetica" SIZE="-1"><A HREF="http://magellan.excite.com/">Home</A> | <A HREF="/magellan/Info/advancedtips.html">Help</A> | <A HREF="/magellan/Info/addsite.html">Add Site</A><P></FONT>
</FORM>
<P>
<HR>
<FONT SIZE=-1>
Explore <A HREF="http://www.excite.com/info/advertising">advertising</A> opportunities with Magellan.<BR>
Copyright &copy;1998 The McKinley Group, Inc., a subsidiary of <A HREF="http://www.excite.com/?Mh-a-t">Excite, Inc.</A>.  All rights reserved.
</FONT>
</CENTER>
<IMG SRC="http://ad.preferences.com/ping.ng;spacedesc=TCExciteData_Excite_1x1_RunOfSite_Any&event=KeywordResults" HEIGHT=1 WIDTH=1 BORDER=0>
</BODY>
</HTML>












