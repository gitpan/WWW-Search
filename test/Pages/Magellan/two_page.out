http://www.isi.norgren.com/
http://www.procite.com/welcome.html
http://www.kfa-juelich.de/isi/isi_e.html
http://www.ast.cam.ac.uk/~ioalib/otherlit.html
http://www.isibrno.cz/~hanca/index.html
http://www.hscc.net/
http://www.isi-bltv.co.uk/ricochet/services.htm
http://www.isibrno.cz/~malczyk/index.html
http://inform.bnl.gov/RESLIB/reslib.html
http://www.hindustantimes.com/
http://www.uni-duisburg.de/duisburg/livetext.htm
http://www.bnl.gov/RESLIB/reslib.html
http://www.afn.org/~iori/siointro.html
http://www.ala.org/acrl/feb97pro.html
http://www.acintl.ro/
http://www-scf.usc.edu/~tejada/
http://www.ucl.ac.uk/~ucylwww/Databases.html
http://www.cedar.buffalo.edu/ShowTell/prototype.html
http://www.isi.edu/sims/arens/index.html
http://www.iub.edu/~cheminfo/ca_chbdi.html
http://www.lib.gla.ac.uk/Subject/Virology/index.html
http://uhura.cc.rochester.edu/~ib001h/
http://wwwcs.newcastle.edu.au/~vlad/kddm.html
http://www.dfki.de/~wahlster/Readings.html
http://www.ianmag.com/Edchoice/0998osci.htm
http://www.scils.rutgers.edu/nj-asis/news.html
http://www.interlog.com/~isi/periodical.html
http://www.fz-juelich.de/auf_einen_blick/auf_einen_blick-e.html
http://www.gbhap-us.com/journals/218/
http://wwwsoc.nacsis.ac.jp/jsme2/link.html
http://chemsources.com/links/other.html
http://www.cacr.caltech.edu/Contact/other_sites.html
http://msrc.wvu.edu/CPMT/index.html
http://www.sun.com/roadtojava/locations/
http://www.opmcorp.com/service/web.html
http://www.library.ucsb.edu/subj/matersci.html
http://www.isi.edu/~touch/pubs/rfc2140.html
http://mscworld.com/education/malaysia.htm
http://thsunl.jinr.dubna.su/~shcher/resume.html
http://www-theory.dcs.st-andrews.ac.uk/~aaa/
http://www.defencejournal.com/feb-mar99/letter.htm
http://www.lib.usf.edu/~ifrank/guides/latinamerica.html
http://www.metronet.com/~rawlins/about.html
http://www.lmu.ac.uk/ebdc/links.htm
http://www.ougf.fi/orainfo/0035.html
http://www.mcbsys.com/html/custlist.html
http://cadlab.cs.ucla.edu/slped96/advprog.html
http://www.colorado.edu/GeolSci/geochem.html
http://www.shef.ac.uk/~lib/subjects/subpgce.html
http://figaro.csee.usf.edu/~vlsi98/vlsi_site/sessions.html
http://www.nedd.com/
http://www.shopquincy.com/golfusa/new.shtml
http://users.aol.com/mmcchesney/resume.html
http://www.singaren.net.sg/apng/in-memoriam.html
http://www.unece.org./stats/links.htm
http://users.aol.com/jcnabity/npgs/links.htm
http://ee10-11.ee.ccu.edu.tw/~whsheen/
http://www.ietf.org/rfc/rfc0937.txt
http://ranier.hq.nasa.gov/Sensors_page/Terms.html
http://www.nrc.gov/OPA/ud021199.htm
http://carmen.umds.ac.uk/d.atkinson/da_bookmarks.html
http://www-sul.stanford.edu/guides/cdroms.html
http://www.ispo.cec.be./infosoc/links.html
http://www.polymers.com/dotcom/all.html
http://www.cs.berkeley.edu/~mccanne/bookmarks.html
http://www.casti.com/RFCs/fyi9.txt
http://wwwhistec.hss.titech.ac.jp/~ktanaka/rfc/rfc1336.txt
http://library.ucf.edu/internet/Poli_Sci/
http://www.startap.net/igrid/index.html
http://jan.mannlib.cornell.edu/jps/jps.htm
http://www.pparc.ac.uk/role/ukastro.html
http://www.phy.hw.ac.uk/~karsten/Bookmarks.html
http://www.bizweb.com/keylists/imaging.html
http://www.beaker.iupui.edu/mla-msls/mslsprograms.html
http://www.linux-howto.com/rfc/rfc1395.txt
http://andrew2.andrew.cmu.edu/rfc/rfc1497.html
http://www.links2go.com/more/dsbb.imf.org/
http://www.babson.edu/library/systems.htm
http://www.sargents.demon.co.uk/Philip/gis-bookmarks.html
http://sci.hirosaki-u.ac.jp/~rmiya/rmiya.bookmarks.html
http://www.amolf.nl/ibmm98/program.html
http://www-me.stanford.edu/WWW-ME/special.html
http://www.matematik.su.se/matstat/annualreport.html
http://stuorg.ucsd.edu/~acm/Jobs/970607.jobs.html
http://www.utep.edu/psych/malpass.html
http://www.rugreview.com/stuf/afgwar.htm
http://www.ifla.org/II/orgs.htm
http://www.library.yale.edu/colldev/prices.html
http://www.lib.umich.edu/libhome/Documents.center/forasia.html
http://www.ietf.org/rfc/rfc0917.txt
http://andrew2.andrew.cmu.edu/rfc/rfc1340.html
http://www.cwu.edu/~warren/sources.html
http://www.cec.sped.org/faq/chromabn.htm
http://www.amolf.nl/ibmm98/proceed.html
http://www.intellworks.com/siren/siren092298.html
http://andrew2.andrew.cmu.edu/rfc/rfc1211.html
http://ihwhp1.bau-verm.uni-karlsruhe.de/~kolleg/10_ckvle.htm
http://www.esofta.com/dev_dir/comp.html
http://www.rsc.anu.edu.au/~harry/INFO/PORSCHE/
http://www.geocities.com/RainForest/1329/LStomatoes.htm
http://www.links2go.com/more/www.stat.unipg.it/
http://www.mathematik.uni-halle.de/links/siteslist.html
http://www.electronicbanker.com/html/archive.htm
http://www.edu.uch.gr/~gts/crete96/participants.html
http://www.snc.edu/library/elecrsrc.htm
http://www.spacelinks.com/SpaceCareers/Careers_Ground.html
http://www.nikhefk.nikhef.nl/~ed/direct/italy.html
http://www.aps.org/BAPSMAR97/tocK.html
http://www.pk.edu.pl/~pmj/dns/dns_faq.txt
http://www.access.digex.net/~ikind/babel.html
http://www.euronet.nl/users/euser/measur.txt
http://www.dsmc.dsm.mil/relsites/rlstsgen.htm
http://wwwbdc.bao.ac.cn/astroweb/cat-radio.html
http://www.nas.edu/cpsma/icfapps.htm
http://www.oralhistory.uconn.edu/~WWWHR/speclpay.html
http://www.cis.upenn.edu/~sjokim/network.txt
http://www.ietf.org/rfc/rfc0997.txt
http://www.mdc.net/~cherokee/florida.txt
http://www2.traveller.com/vdoms/index.html
http://library.fortlewis.edu/~online/descript.html
http://www.umaryland.edu/hshsl/colldev.html
http://www.uab.edu/lister/databases.htm
http://www.linux-howto.com/rfc/rfc787.txt
http://www.rsl.ait.ac.th/~vivarad/GISCat/abbrev.html
http://www.library.arizona.edu/users/arawan/descript.htm
http://ethics.engl.cwru.edu/bib/newbib.html
http://www.vilspa.esa.es/astroweb/yp_optical.html
http://www.cogsci.ed.ac.uk/~simone/AITS/chapter.html
http://www.teleport.com./~glowniak/babel95c.txt
http://www.lb.cs.cmu.edu/~softagents/DAIlist/0032.html
http://www.ankara.edu.tr/~astro/AstroResource/telescope.html
http://www.mysql.com/Manual_chapter/manual_News.html
http://www.cis.nctu.edu.tw/~is84086/NetProg/rfc959.txt
http://www-nishio.ise.eng.osaka-u.ac.jp/~shirai/research/babel.html
http://www.lib.berkeley.edu/LBO/vendcode.html
http://biz.yahoo.com./news/computer.html
http://osiris.rutgers.edu/~zoli/works/help.html
http://www.sissa.it/~furio/journal_sort.html
http://www.cis.ohio-state.edu/hypertext/faq/usenet/alt-usage-english-faq/faq.html
http://www.linux-howto.com/rfc/rfc1020.txt
